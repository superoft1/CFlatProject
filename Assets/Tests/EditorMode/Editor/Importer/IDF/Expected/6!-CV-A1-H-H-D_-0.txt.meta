fileFormatVersion: 2
guid: db47d2d35f14622498fafe09f2e7bbc7
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
