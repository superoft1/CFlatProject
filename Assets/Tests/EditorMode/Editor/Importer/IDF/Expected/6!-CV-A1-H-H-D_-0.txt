Elbow90, , (0.0, 0.0, -1.0), (-789.1, 93.0, 1.5), False
(-789.1, 93.3, 1.5)
(-789.1, 93.0, 1.7)
FreeVertex: 1
Pipe, , (0.0, 0.0, 1.0), (-789.1, 93.0, 2.1), False
(-789.1, 93.0, 1.7)
(-789.1, 93.0, 2.4)
FreeVertex: 0
Elbow90, , (1.0, 0.0, 0.0), (-789.1, 93.0, 2.6), False
(-789.1, 93.0, 2.4)
(-789.3, 93.0, 2.6)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-789.4, 93.0, 2.6), False
(-789.3, 93.0, 2.6)
(-789.5, 93.0, 2.6)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-789.7, 93.0, 2.6), False
(-789.5, 93.0, 2.6)
(-789.8, 93.0, 2.6)
FreeVertex: 0
WeldNeckFlange, , (-1.0, 0.0, 0.0), (-789.8, 93.0, 2.6), False
(-789.9, 93.0, 2.6)
(-789.8, 93.0, 2.6)
FreeVertex: 0
GateValve, , (-1.0, 0.0, 0.0), (-790.0, 93.0, 2.6), False
(-790.2, 93.0, 2.6)
(-789.9, 93.0, 2.6)
FreeVertex: 0
WeldNeckFlange, , (1.0, 0.0, 0.0), (-790.2, 93.0, 2.6), False
(-790.2, 93.0, 2.6)
(-790.2, 93.0, 2.6)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-790.3, 93.0, 2.6), False
(-790.2, 93.0, 2.6)
(-790.4, 93.0, 2.6)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-790.4, 93.0, 2.6), False
(-790.4, 93.0, 2.6)
(-790.4, 93.0, 2.6)
(-790.4, 93.0, 2.5)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-790.4, 93.0, 2.5), False
(-790.4, 93.0, 2.5)
(-790.4, 93.0, 2.5)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-790.4, 93.0, 2.4), False
(-790.4, 93.0, 2.4)
(-790.4, 93.0, 2.5)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-790.4, 93.0, 2.4), False
(-790.4, 93.0, 2.4)
(-790.4, 93.0, 2.3)
FreeVertex: 0
Cap, , (0.0, 0.0, -1.0), (-790.4, 93.0, 2.3), False
(-790.4, 93.0, 2.3)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-790.5, 93.0, 2.6), False
(-790.4, 93.0, 2.6)
(-790.5, 93.0, 2.6)
FreeVertex: 0
EccentricPipingReducer, , (1.0, 0.0, 0.0), (-790.6, 93.0, 2.6), False
(-790.5, 93.0, 2.6)
(-790.7, 93.0, 2.7)
FreeVertex: 0
WeldNeckFlange, , (-1.0, 0.0, 0.0), (-790.7, 93.0, 2.7), False
(-790.8, 93.0, 2.7)
(-790.7, 93.0, 2.7)
FreeVertex: 0
DiaphramControlValve, FCV-2202, (-1.0, 0.0, 0.0), (-790.9, 93.0, 2.7), False
(-791.1, 93.0, 2.7)
(-790.8, 93.0, 2.7)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-791.1, 93.0, 2.7), False
(-791.1, 93.0, 2.7)
(-791.1, 93.0, 2.7)
FreeVertex: 0
WeldNeckFlange, , (1.0, 0.0, 0.0), (-791.2, 93.0, 2.7), False
(-791.1, 93.0, 2.7)
(-791.2, 93.0, 2.7)
FreeVertex: 0
EccentricPipingReducer, , (-1.0, 0.0, 0.0), (-791.3, 93.0, 2.6), False
(-791.3, 93.0, 2.6)
(-791.2, 93.0, 2.7)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-791.4, 93.0, 2.6), False
(-791.3, 93.0, 2.6)
(-791.5, 93.0, 2.6)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-791.5, 93.0, 2.6), False
(-791.5, 93.0, 2.6)
(-791.5, 93.0, 2.6)
(-791.5, 93.0, 2.5)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-791.5, 93.0, 2.5), False
(-791.5, 93.0, 2.5)
(-791.5, 93.0, 2.5)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-791.5, 93.0, 2.4), False
(-791.5, 93.0, 2.4)
(-791.5, 93.0, 2.5)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-791.5, 93.0, 2.4), False
(-791.5, 93.0, 2.4)
(-791.5, 93.0, 2.3)
FreeVertex: 0
Cap, , (0.0, 0.0, -1.0), (-791.5, 93.0, 2.3), False
(-791.5, 93.0, 2.3)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-791.6, 93.0, 2.6), False
(-791.5, 93.0, 2.6)
(-791.6, 93.0, 2.6)
FreeVertex: 0
WeldNeckFlange, , (-1.0, 0.0, 0.0), (-791.7, 93.0, 2.6), False
(-791.7, 93.0, 2.6)
(-791.6, 93.0, 2.6)
FreeVertex: 0
GateValve, , (-1.0, 0.0, 0.0), (-791.9, 93.0, 2.6), False
(-792.0, 93.0, 2.6)
(-791.7, 93.0, 2.6)
FreeVertex: 0
WeldNeckFlange, , (1.0, 0.0, 0.0), (-792.0, 93.0, 2.6), False
(-792.0, 93.0, 2.6)
(-792.1, 93.0, 2.6)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-792.2, 93.0, 2.6), False
(-792.1, 93.0, 2.6)
(-792.3, 93.0, 2.6)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-792.5, 93.0, 2.6), False
(-792.3, 93.0, 2.6)
(-792.6, 93.0, 2.6)
FreeVertex: 0
Elbow90, , (0.0, 0.0, 1.0), (-792.8, 93.0, 2.6), False
(-792.6, 93.0, 2.6)
(-792.8, 93.0, 2.4)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-792.8, 93.0, 2.1), False
(-792.8, 93.0, 2.4)
(-792.8, 93.0, 1.7)
FreeVertex: 0
Elbow90, , (0.0, -1.0, 0.0), (-792.8, 93.0, 1.5), False
(-792.8, 93.0, 1.7)
(-792.8, 93.3, 1.5)
FreeVertex: 1
FreeVertexTotal: 2
